components/user/UserMilestone.js

```diff
@@ -12,12 +12,14 @@ export default function UserMilestone({ milestone, isGoal, manage }) {
   const [date, setDate] = useState(milestone.date);
 
   useEffect(() => {
-    const formattedDate = shortenDate({
-      date: milestone.date,
-      formatStyle: milestone.dateFormat,
-    });
-    setDate(formattedDate);
-  }, [milestone, milestone.dateFormat]);
+    if(milestone.date) {
```

Comment:
Please check you have the VScode plugin for Prettier to auto fix any formatting issues on save

```suggestion
    if (milestone.date) {
```